tools/mo/unit_tests/mo/convert/logger_test_actual.py

```diff
@@ -26,7 +26,7 @@ def create_tf_model(out_dir):
 
 
 def run_main():
-    from openvino.tools.mo import convert_model
+    from openvino.runtime import convert_model
```

Comment:
the same question. May be we should leave this test without changes? And move it to OVC unit-tests after?

Reply:
Removed this change.